engine/src/main/java/org/terasology/rendering/dag/nodes/AmbientOcclusionNode.java

```diff
@@ -112,8 +113,8 @@ public AmbientOcclusionNode(Context context) {
         ssaoFbo = requiresFBO(new FBOConfig(SSAO_FBO_URI, FULL_SCALE, FBO.Type.DEFAULT), displayResolutionDependentFBOs);
         addDesiredStateChange(new BindFbo(ssaoFbo));
         addDesiredStateChange(new SetViewportToSizeOf(ssaoFbo));
-        update(); // Cheeky way to initialise outputFboWidth, outputFboHeight
-        displayResolutionDependentFBOs.subscribe(this);
+        propertyChange(null); // Cheeky way to initialise outputFboWidth, outputFboHeight
+        displayResolutionDependentFBOs.subscribe(DisplayResolutionDependentFBOs.POST_FBO_REGENERATION, this);
```

Comment:
Let's remove the `DisplayResolutionDependentFBOs.` part please. Had it been a short string we could have left it, like we do with FBO.HDR for example. But in this case the most important information is the event type and POST_FBO_REGENERATION gives a good amount of information already.